File: boletim-ocorrencia/src/main/java/br/edu/utfpr/td/tsi/sistema/boletim/ocorrencia/model/StolenVehicleReport.java
package br.edu.utfpr.td.tsi.sistema.boletim.ocorrencia.model;

import java.time.LocalDate;

public class StolenVehicleReport {

    private Integer id;
    private LocalDate occurrenceDate;
    private String occurrenceDayTime;
    private Part part;
    private Address occurrenceLocal;
    private Vehicle stolenVehicle;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public LocalDate getOccurrenceDate() {
        return occurrenceDate;
    }

    public void setOccurrenceDate(LocalDate occurrenceDate) {
        this.occurrenceDate = occurrenceDate;
    }

    public String getOccurrenceDayTime() {
        return occurrenceDayTime;
    }

    public void setOccurrenceDayTime(String occurrencePeriod) {
        this.occurrenceDayTime = occurrencePeriod;
    }

    public Part getPart() {
        return part;
    }

    public void setPart(Part part) {
        this.part = part;
    }

    public Address getOccurrenceLocal() {
        return occurrenceLocal;
    }

    public void setOccurrenceLocal(Address occurrenceLocal) {
        this.occurrenceLocal = occurrenceLocal;
    }

    public Vehicle getStolenVehicle() {
        return stolenVehicle;
    }

    public void setStolenVehicle(Vehicle stolenVehicle) {
        this.stolenVehicle = stolenVehicle;
    }
}

File: boletim_python_v/occurance_report_system/occurance_report_system/occurance_report/models/stolen_vehicle_report.py
from .part import Part
from .address import Address
from .vehicle import Vehicle

class StolenVehicleReport:
    def __init__(self):
        self.id = None
        self.occurrence_date = None
        self.occurrence_day_time = None
        self.part = None
        self.occurrence_local = None
        self.stolen_vehicle = None

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_occurrence_date(self):
        return self.occurrence_date

    def set_occurrence_date(self, occurrence_date):
        self.occurrence_date = occurrence_date

    def get_occurrence_day_time(self):
        return self.occurrence_day_time

    def set_occurrence_day_time(self, occurrence_day_time):
        self.occurrence_day_time = occurrence_day_time

    def get_part(self):
        return self.part

    def set_part(self, part):
        self.part = part

    def get_occurrence_local(self):
        return self.occurrence_local

    def set_occurrence_local(self, occurrence_local):
        self.occurrence_local = occurrence_local

    def get_stolen_vehicle(self):
        return self.stolen_vehicle

    def set_stolen_vehicle(self, stolen_vehicle):
        self.stolen_vehicle = stolen_vehicle
